extensions/mssql/src/objectManagement/commands.ts

```diff
@@ -106,15 +107,19 @@ async function handleObjectPropertiesDialogCommand(context: azdata.ObjectExplore
 		return;
 	}
 	try {
-		const parentUrn = await getParentUrn(context);
+		const parentUrn = context.nodeInfo ? await getParentUrn(context) : undefined;
```

Comment:
I haven't tested but are you sure the nodeInfo is undefined for server node context?

Reply:
Please sync up with @ssreerama , I left a comment in this about adding logic to handle this in getParentUrn.

Given that you're both using it I would even suggest sending out a quick separate PR for changing this, that way you can both merge it in and use it without needing to wait on the other to finish.

Reply:
@barbaravaldez Did you get a chance to do that? You can do it as a follow up item if you want, it's not a huge deal. But it'd be good to make the behavior consistent for anyone using it.